cmake/common.rc

```diff
@@ -0,0 +1,7 @@
+/*
+MinGW can't compile multiple .rc files. Only single.
```

Comment:
Perhaps a better fix, simpler for MinGW, would be to include `translations.rc` from `winsparkle.rc` then?